.circleci/config.yml

```diff
@@ -7,16 +7,16 @@ jobs:
       - run: | 
           TAG=0.1.$CIRCLE_BUILD_NUM
           docker login -u $DOCKER_USER -p $DOCKER_PASS
-          docker build -t ajaytripathy/kubecost-cost-model:$TAG .
-          docker push ajaytripathy/kubecost-cost-model:$TAG
+          docker build -t kubecost/cost-model:$TAG .
+          docker push kubecost/cost-model:$TAG
   deploy:
     machine: true
     steps:
       - checkout
       - run: |
           docker login -u $DOCKER_USER -p $DOCKER_PASS
-          docker build -t ajaytripathy/kubecost-cost-model:latest .
-          docker push ajaytripathy/kubecost-cost-model:latest
+          docker build -t kubecost/cost-model:latest .
```

Comment:
And throughout: quay.io/kubecost1/kubecost-cost-model:latest

Reply:
updated. we might need to see an actual run, I'm not sure that `docker build` allows for `quay.io` prefix